qualtran/cirq_interop/_cirq_interop.py

```diff
@@ -135,7 +144,7 @@ def wire_symbol(self, soq: 'Soquet') -> 'WireSymbol':
         begin = 0
         symbol: str = soq.pretty()
         for reg in self.signature:
-            finish = begin + np.product(reg.shape)
+            finish = begin + int(np.product(reg.shape))
```

Comment:
is the int cast here and above necessary?

Reply:
Yes, `np.product` returns a float instead of an `int` so the cast is necessary. I should probably also replace `np.product` with `np.prod` instead. Will do that.